fanctrl.py

```diff
@@ -138,7 +138,7 @@ def updateTemperature(self):
         sumCoreTemps = 0
         sensorsOutput = json.loads(
             subprocess.run(
-                "sensors -j",
+                "sensors -j &> /dev/null",
```

Comment:
This should be `sensors -j 2> /dev/null`

Reply:
Honestly, I am sending all to null here, I don't want anything returned or dumped to console or logs.